LDAR_Sim/src/file_processing/output_processing/summary_visualization_mapper.py

```diff
@@ -0,0 +1,83 @@
+from matplotlib import ticker
+from file_processing.output_processing import output_utils
+from file_processing.output_processing.output_utils import percent_difference, relative_difference
+from constants import output_file_constants
+
+
+class SummaryVisualizationMapper:
+    def __init__(self):
+        self._summary_stat_functions = {
+            (
+                output_file_constants.SummaryVisualizationStatistics.PERCENT_DIFFERENCE
+            ): percent_difference,
+            (
+                output_file_constants.SummaryVisualizationStatistics.RELATIVE_DIFFERENCE
+            ): relative_difference,
+        }
+
+        self._histogram_properties_lookup = {
+            output_file_constants.FileNames.TRUE_VS_ESTIMATED_PERCENT_DIFF_PLOT: {
+                "hist0": {
+                    "bin_width": (
+                        (
+                            output_file_constants.ESTIMATE_VS_TRUE_PLOTTING_CONSTANTS
+                        ).HISTOGRAM_BIN_WIDTH
+                    ),
+                    "bin_range": (
+                        output_file_constants.ESTIMATE_VS_TRUE_PLOTTING_CONSTANTS
+                    ).HISTOGRAM_PERCENT_DIFF_BIN_RANGE,
+                    "x_label": output_file_constants.HistogramConstants.X_AXIS_LABEL_PERCENT,
+                    "y_label": output_file_constants.HistogramConstants.Y_AXIS_LABEL,
+                }
+            },
+            output_file_constants.FileNames.TRUE_VS_ESTIMATED_RELATIVE_DIFF_PLOT: {
+                "hist0": {
+                    "bin_width": (
+                        (
+                            output_file_constants.ESTIMATE_VS_TRUE_PLOTTING_CONSTANTS
+                        ).HISTOGRAM_BIN_WIDTH
+                    ),
+                    "bin_range": (-100, None),
+                    "x_label": output_file_constants.HistogramConstants.X_AXIS_LABEL_RELATIVE,
+                    "y_label": output_file_constants.HistogramConstants.Y_AXIS_LABEL,
+                    "x_locator": ticker.MaxNLocator(nbins="auto", prune=None),
+                }
+            },
+            output_file_constants.FileNames.TRUE_AND_ESTIMATED_PAIRED_EMISSIONS_DISTRIBUTION_PLOT: {
+                "hist0": {
+                    "x_label": output_file_constants.HistogramConstants.X_AXIS_LABEL_PAIRED,
+                    "y_label": output_file_constants.HistogramConstants.Y_AXIS_LABEL,
+                    "bins": output_file_constants.HistogramConstants.BINS,
+                    "bin_range": (None, None),
+                    "legend_label": (
+                        output_file_constants.ESTIMATE_VS_TRUE_PLOTTING_CONSTANTS
+                    ).TRUE_EMISSIONS_SUFFIX,
+                },
+                "hist1": {
+                    "x_label": output_file_constants.HistogramConstants.X_AXIS_LABEL_PAIRED,
```

Comment:
This label requires formatting before use -> "Annual Emissions of All {n_sites} Sites (Kg Methane)"
The n_sites need to be set.